ansible/playbooks/adhoc/bootstrap-fedora.yml

```diff
@@ -0,0 +1,4 @@
+- hosts: OSEv3
```

Comment:
This file should be migrated to openshift-ansible-contrib rather than openshift-tools